test/e2e/cluster_samples_operator_test.go

```diff
@@ -592,6 +595,9 @@ func verifyDeletedImageStreamNotRecreated(t *testing.T) {
 	if !verifyX86(t) {
 		return
 	}
+	if util.IPv6Only() {
+		return
+	}
```

Comment:
This could be interesting...I suspect that the test runner is on our CI server. No idea how the ipv6 clusters communicate with the rest of the world if they are truly single-stack (rather than dualstack).

Another reason why this info should be exposed to the cluster vs. our hacks.

Reply:
Good point.

To date the working doc has `you can look at the network configuration, either in install-config.yaml for things that run at bootstrap time, or in the “cluster” network.config.openshift.io object later on. eg, look at the entries in spec.ServiceNetwork and if they are all IPv6, then you are on a single-stack IPv6 cluster. `

I'll take a look at network object and follow up in forum-ipv6 and then circle back to the PR, at least for the test runner.

I think we are OK with the in-operator stuff.

Reply:
ok I've pushed the update that leverages the cluster network config to determine ipv4 vs. ipv6 for the framework

to clarify on why we should use the k8s utility within the operator, it checks the pod locally and avoids an RPC call to the api server.

PTAL @adambkaplan